src/Class.js

```diff
@@ -1,10 +1,10 @@
 var WebJack = {};
 
-(function(exports){
```

Comment:
er, that is, to make it buildable with browserify AND usable directly in a browser